### PR TITLE
experimental CLI documentation

### PR DESCRIPTION
**Pull request by wdbaruni**
ENG-131

### CODE REVIEW

File: docs/references/cli/agent/node/index.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- `-h`, `--help`:
+  - Displays help information for the `node` sub-command.
+
+- `--output format`:
+  - Defines the output format (either JSON or YAML).
+  - Options: `json`, `yaml`
+  - Default: `yaml`
+
+- `--pretty`:
+  - Beautifies the output when using JSON or YAML formats.
```
Comment (rossjones) [inline docs/references/cli/agent/node/index.md:27]: Not really relevant to this PR, but this should probably be the default.

File: docs/references/cli/agent/version/index.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+sidebar_label: version
+---
+# Command: `agent version
```
Comment (rossjones) [inline docs/references/cli/agent/version/index.md:4]: missing trailing `

File: docs/references/cli/agent/version/index.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+The `bacalhau agent version` command is utilized to obtain the version of the bacalhau agent.
```
Comment (rossjones) [inline docs/references/cli/agent/version/index.md:6]:
```suggestion
The `bacalhau agent version` command is used to obtain the version of the bacalhau agent.
```

File: docs/references/cli/index.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Commands CLI (Experimental)
+
+:::caution
+The following commands refer to experimental bacalhau cli version `v1.1.0`. Experimental means that the commands are not stable and may change in future versions. More stable, with limited functionality commands are documented [here](/all-flags).
```
Comment (rossjones) [inline docs/references/cli/index.md:4]:
```suggestion
The following commands refer to experimental bacalhau cli version `v1.1.0`. Experimental means that the commands are not stable and may change in future versions. More stable commands, with limited functionality, are documented [here](/all-flags).
```

File: docs/references/cli/agent/node/index.md (hunk 1 above)
Comment (wdbaruni) [reply docs/references/cli/agent/node/index.md:27]: That makes sense. Opened a backlog item https://expanso.atlassian.net/browse/ENG-155

File: docs/references/cli/agent/version/index.md (hunk 2 above)
Comment (wdbaruni) [reply docs/references/cli/agent/version/index.md]: fixed in https://github.com/bacalhau-project/docs.bacalhau.org/pull/274/commits/ced020632c47375367bb0ab27b9494e80afbd1a2

**Review by rossjones — APPROVED** [review]

File: docs/references/cli/agent/node/index.md (hunk 1 above)
Comment (rossjones) [reply docs/references/cli/agent/node/index.md:27]: Me too :D

**Pull request merged**